src/parse-env.js

```diff
@@ -0,0 +1,32 @@
+module.exports = env => {
+  const {
+    AIRTABLE_BASE_ID,
+    AIRTABLE_API_KEY,
+    ALLOWED_METHODS,
+    PORT = 3000,
+    READ_ONLY,
+  } = env;
+
+  if (
+    typeof AIRTABLE_BASE_ID === 'undefined' ||
+    typeof AIRTABLE_API_KEY === 'undefined'
+  ) {
+    throw new TypeError(
+      'Please provide AIRTABLE_BASE_ID and AIRTABLE_API_KEY as environment variables.'
```

Comment:
If these variables are missing, they'd also be caught by the call to the handler function. I opted to keep it here to have a more specific and actionable error message. Namely, saying that they're missing env variables.

There's potentially more things we'd want to catch here.